deploy/charts/metrics-agent/Chart.yaml

```diff
@@ -14,8 +14,8 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 0.1.0
+version: 0.2.0
```

Comment:
I noticed in the Makefile the Helm version uses the release version. Should it be this version?

https://github.com/cloudability/metrics-agent/blob/bbeb62dc81ce36a65f10a21cdc1d35aeeb3d3d56/Makefile#L58

Reply:
Good catch. I took the hardcoded versions out of the `helm package` command in the Makefile. Not sure why we would need that, vs. just sticking with the versions that are in the chart.